### PR TITLE
Feature/20 Registration Service API ADR

### PR DESCRIPTION
**Pull request by Izzzu**
Initial ADR in Registration Service describing the Registration Service API.

Linked to https://github.com/agera-edc/MinimumViableDataspace/issues/20

### CODE REVIEW

File: docs/developer/decision-records/2022-06-13-registration-service-api/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of Dataspace Authority.
```
Comment (chrislomonico) [inline docs/developer/decision-records/2022-06-13-registration-service-api/README.md]:
```suggestion
Registration Service is a component of a Dataspace Authority. 
```

File: docs/developer/decision-records/2022-06-13-registration-service-api/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of a Dataspace Authority. 
+One of the responsibilities of the Registration Service is a management of the participants in the Dataspace.
```
Comment (chrislomonico) [inline docs/developer/decision-records/2022-06-13-registration-service-api/README.md]:
```suggestion
One of the responsibilities of the Registration Service is the management of the participants in the Dataspace.
```

File: docs/developer/decision-records/2022-06-13-registration-service-api/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of a Dataspace Authority. 
+One of the responsibilities of the Registration Service is the management of the participants in the Dataspace.
+
+The Registration Service exposes an API that offers following operations:
```
Comment (chrislomonico) [inline docs/developer/decision-records/2022-06-13-registration-service-api/README.md]:
```suggestion
The Registration Service exposes an API that offers the following operations:
```

File: docs/developer/decision-records/2022-06-13-registration-service-api/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of a Dataspace Authority. 
+One of the responsibilities of the Registration Service is the management of the participants in the Dataspace.
+
+The Registration Service exposes an API that offers the following operations:
+- Add participant to the Dataspace
```
Comment (chrislomonico) [inline docs/developer/decision-records/2022-06-13-registration-service-api/README.md]:
```suggestion
- Add a participant to the Dataspace
```

**Review by chrislomonico — APPROVED** [review]
A few small text changes already submitted

File: docs/developer/decision-records/2022-06-13-registration-service-api/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+The Registration Service is written in Java and uses the runtime framework and modules from [EDC](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector).
+
+## OpenAPI 
+
+The Registration Service API definition is available in an [Open API yaml file](../../../../resources/openapi/yaml/registration-service.yaml). 
+
+## API Rest client
+
+The Registration Service API can be called using the [rest-client](../../../../rest-client) generated from OpenApi definition.
```
Comment (algattik) [inline docs/developer/decision-records/2022-06-13-registration-service-api/README.md:18]: make spelling uniform "open API" "OpenApi"

**Review by algattik — APPROVED** [review]

**Review by cpeeyush — APPROVED** [review]

**Pull request merged**